test/parallel/test-http-dump-req-when-res-ends.js

```diff
@@ -51,4 +52,13 @@ server.listen(0, function() {
       server.close();
     });
   }));
+
+  req.on('error', function(err) {
+    // ECONNRESET can happen if there is some data still
+    // being sent, as the other side is calling .destroy()
+    if (err.code !== 'ECONNRESET') {
```

Comment:
I _think_ on Windows it's `ECONNABORTED`, and it might also be `EPIPE` as per https://github.com/nodejs/node/issues/19139#issue-302131586. Should we just ignore the error?